go/libkb/secret_store_external.go

```diff
@@ -7,6 +7,10 @@ package libkb
 
 import "sync"
 
+// TODO: Make this implementation use GetStoredSecretServiceName(), as
```

Comment:
Chatted with @MarcoPolo, for Android, we'll just concatenate the service name with the account name (with a suitable separator) and use that as the 'username' for now, since Android just has a single key-value store.

Another possibility is to encode both using msgpack. That might be more robust? We'll see.

I'll do that in a follow-up PR to this one.